ITC-2: Some packages could not be installed

from https://github.com/rsmp-nordic/rsmp_validator/actions/runs/5052931260/jobs/9066269254:

```
Run sudo dpkg --add-architecture i386
  sudo dpkg --add-architecture i386
  sudo apt-get update
  sudo apt-get -y install wine32
  shell: /usr/bin/bash -e {0}
Hit:1 http://azure.archive.ubuntu.com/ubuntu jammy InRelease
Hit:2 http://azure.archive.ubuntu.com/ubuntu jammy-updates InRelease
Hit:3 http://azure.archive.ubuntu.com/ubuntu jammy-backports InRelease
Hit:4 http://azure.archive.ubuntu.com/ubuntu jammy-security InRelease
Hit:5 https://packages.microsoft.com/ubuntu/22.04/prod jammy InRelease
Get:6 http://azure.archive.ubuntu.com/ubuntu jammy/main i386 Packages [1040 kB]
Get:7 http://azure.archive.ubuntu.com/ubuntu jammy/restricted i386 Packages [30.4 kB]
Get:8 http://azure.archive.ubuntu.com/ubuntu jammy/universe i386 Packages [7474 kB]
Get:9 http://azure.archive.ubuntu.com/ubuntu jammy/multiverse i386 Packages [112 kB]
Get:10 http://azure.archive.ubuntu.com/ubuntu jammy-updates/main i386 Packages [393 kB]
Get:11 http://azure.archive.ubuntu.com/ubuntu jammy-updates/restricted i386 Packages [28.3 kB]
Get:12 http://azure.archive.ubuntu.com/ubuntu jammy-updates/universe i386 Packages [611 kB]
Get:13 http://azure.archive.ubuntu.com/ubuntu jammy-updates/multiverse i386 Packages [3884 B]
Hit:14 https://ppa.launchpadcontent.net/ubuntu-toolchain-r/test/ubuntu jammy InRelease
Get:15 http://azure.archive.ubuntu.com/ubuntu jammy-backports/main i386 Packages [33.7 kB]
Get:16 http://azure.archive.ubuntu.com/ubuntu jammy-backports/universe i386 Packages [12.7 kB]
Get:17 http://azure.archive.ubuntu.com/ubuntu jammy-security/main i386 Packages [212 kB]
Get:18 http://azure.archive.ubuntu.com/ubuntu jammy-security/restricted i386 Packages [28.0 kB]
Get:19 http://azure.archive.ubuntu.com/ubuntu jammy-security/universe i386 Packages [529 kB]
Get:20 http://azure.archive.ubuntu.com/ubuntu jammy-security/multiverse i386 Packages [1028 B]
Get:21 https://ppa.launchpadcontent.net/ubuntu-toolchain-r/test/ubuntu jammy/main i386 Packages [2124 B]
Fetched 10.5 MB in 2s (6384 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libc6:i386 : Depends: libgcc-s1:i386 but it is not going to be installed
 libicu70:i386 : Depends: libgcc-s1:i386 (>= 7) but it is not going to be installed
                 Depends: libstdc++6:i386 (>= 5.2) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
Error: Process completed with exit code 100.
```

--- a/.github/workflows/swarco_itc2.yml
+++ b/.github/workflows/swarco_itc2.yml
@@ -18,12 +18,10 @@ jobs:
 
       - name: Install wine32
         run: |
+          sudo rm -f /etc/apt/sources.list.d/microsoft-prod.list
           sudo dpkg --add-architecture i386
-          sudo mkdir -p /etc/apt/keyrings
-          sudo wget -O /etc/apt/keyrings/winehq-archive.key https://dl.winehq.org/wine-builds/winehq.key
-          sudo wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/ubuntu/dists/$(lsb_release -sc)/winehq-$(lsb_release -sc).sources
-          sudo apt-get update
-          sudo apt-get -y install --install-recommends winehq-stable
+          sudo apt-get update -qq
+          sudo apt-get install -yqq --allow-downgrades libc6:i386 libgcc-s1:i386 libstdc++6:i386 wine
 
       - name: Checkout repo
         uses: actions/checkout@v2

--- a/.github/workflows/swarco_itc2.yml
+++ b/.github/workflows/swarco_itc2.yml
@@ -9,7 +9,7 @@ on:
     - cron:  '45 0 * * *'
 jobs:
   test:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
       - name: Install prerequisites
         run: |

--- a/.github/workflows/swarco_itc2.yml
+++ b/.github/workflows/swarco_itc2.yml
@@ -9,7 +9,7 @@ on:
     - cron:  '45 0 * * *'
 jobs:
   test:
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-20.04
     steps:
       - name: Install prerequisites
         run: |

--- a/.github/workflows/swarco_itc2.yml
+++ b/.github/workflows/swarco_itc2.yml
@@ -19,8 +19,8 @@ jobs:
       - name: Install wine32
         run: |
           sudo dpkg --add-architecture i386
-          wget -nc https://dl.winehq.org/wine-builds/winehq.key
-          sudo mv winehq.key /usr/share/keyrings/winehq-archive.key
+          sudo wget -O /etc/apt/keyrings/winehq-archive.key https://dl.winehq.org/wine-builds/winehq.key
+          sudo wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/ubuntu/dists/jammy/winehq-jammy.sources
           wget -nc https://dl.winehq.org/wine-builds/ubuntu/dists/jammy/winehq-jammy.sources
           sudo mv winehq-jammy.sources /etc/apt/sources.list.d/
           sudo apt-get update

--- a/.github/workflows/swarco_itc2.yml
+++ b/.github/workflows/swarco_itc2.yml
@@ -19,12 +19,11 @@ jobs:
       - name: Install wine32
         run: |
           sudo dpkg --add-architecture i386
+          sudo mkdir -p /etc/apt/keyrings
           sudo wget -O /etc/apt/keyrings/winehq-archive.key https://dl.winehq.org/wine-builds/winehq.key
-          sudo wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/ubuntu/dists/jammy/winehq-jammy.sources
-          wget -nc https://dl.winehq.org/wine-builds/ubuntu/dists/jammy/winehq-jammy.sources
-          sudo mv winehq-jammy.sources /etc/apt/sources.list.d/
+          sudo wget -NP /etc/apt/sources.list.d/ https://dl.winehq.org/wine-builds/ubuntu/dists/$(lsb_release -sc)/winehq-$(lsb_release -sc).sources
           sudo apt-get update
-          sudo apt-get -y install wine32
+          sudo apt-get -y install --install-recommends winehq-stable
 
       - name: Checkout repo
         uses: actions/checkout@v2

--- a/.github/workflows/swarco_itc2.yml
+++ b/.github/workflows/swarco_itc2.yml
@@ -19,6 +19,10 @@ jobs:
       - name: Install wine32
         run: |
           sudo dpkg --add-architecture i386
+          wget -nc https://dl.winehq.org/wine-builds/winehq.key
+          sudo mv winehq.key /usr/share/keyrings/winehq-archive.key
+          wget -nc https://dl.winehq.org/wine-builds/ubuntu/dists/jammy/winehq-jammy.sources
+          sudo mv winehq-jammy.sources /etc/apt/sources.list.d/
           sudo apt-get update
           sudo apt-get -y install wine32
 

